src/layout/layout.rs

```diff
@@ -530,7 +605,7 @@ impl Layout {
         self.split_with_spacers(area).0
     }
 
-    /// Wrapper function around the cassowary-r solver that splits the given area into smaller ones
+    /// Wrapper function around the cassowary-rs solver that splits the given area into smaller ones
```

Comment:
We can actually drop the `-rs` suffix here since the library is called "cassowary".

Reply:
Made an issue since it might be a good first issue for someone intending to contribute - #1423